blueprints/data-solutions/shielded-folder/log-export.tf

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# tfdoc:file:description Audit log project and sink.
+
+locals {
+  gcs_storage_class = (
+    length(split("-", var.log_locations.storage)) < 2
+    ? "MULTI_REGIONAL"
+    : "REGIONAL"
+  )
+  log_types = toset([for k, v in var.log_sinks : v.type])
+
+  _log_keys = var.enable_features.encryption ? {
+    bq      = var.enable_features.log_sink ? ["projects/${module.sec-project.0.project_id}/locations/${var.log_locations.bq}/keyRings/${var.log_locations.bq}/cryptoKeys/bq"] : null
```

Comment:
we need to find a way of breaking these long lines, I can try after you merge

Reply:
Thanks